### DISCUSSION TITLE
Acceleration Range

### DISCUSSION THREAD

**Discussion by susovanpatra00** [jrowberg/i2cdevlib#769] [Q&A]
I am using the code - " Examples --> MPU6050 --> MPU6050_DMP6"
And inside that I am only using "#define OUTPUT_READABLE_REALACCEL"
I want to know the range of this. Is it +-2g or +-4g or something else. What I was doing is I was dividing the value with 16384 (As I thought the range is +-2g, but for some I am getting values greater than 2, So I am confused what to do).

Please Help!!

**Comment by jrowberg** [comment]
I believe the default DMP behavior is +/- 2g, but this may not be correct (or it might vary depending on which version of the DMP binary blob is loaded into the IMU).

I recommend that you test by modifying the `Serial.print` calls so they output what is in the unmodified `aa` struct instead of the `aaReal` struct, as the first one still includes the gravity component. If you set the IMU down flat on a stable surface, you should end up with two `aa` axes that are near zero and one that is near 1g, which should obviously be around 16384, 8192, 4096, or 2048 depending on the sensitivity used in the active DMP configuration.
